### PR TITLE
Add EIP: Reserved Ownership Accounts

### PR DESCRIPTION
**Pull request by sullivph**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

File: EIPS/eip-6981.md [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+    function _getSalt(address deployer, uint96 index) private pure returns (bytes32) {
+        return bytes32(abi.encodePacked(deployer, index));
+    }
+}
+```
+
+### Account Registry
+
+```solidity
+// SPDX-License-Identifier: MIT
```
Comment (SamWilsn) [inline EIPS/eip-6981.md:182]: If the solidity is included inline in the proposal, it must have a license of CC0-1.0

Comment (sullivph) [reply EIPS/eip-6981.md]: Updated!

File: EIPS/eip-6981.md [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-04-25
+requires: 1167, 1271
+---
+
+## Abstract
+
+The following specifies a system for services to provide their users with a deterministic wallet address tied to identifying credentials (e.g. email address, phone number, etc.) without any blockchain interaction, and a mechanism to deploy and control a smart contract wallet (Account Instance) at the deterministic address in the future.
```
[the remark this answers is not in the document]
Comment (sullivph) [reply EIPS/eip-6981.md:16]: Added a `Usage Example` section that illustrates how a service would use this proposal to provide ROAs to their users

**Review by xlansmasry — APPROVED** [review]

File: EIPS/eip-6981.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (SamWilsn) [reply EIPS/eip-6981.md:16]: Oh, sorry, I meant specifically to the abstract, to satisfy EIP-1:

> Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the specification section. Someone should be able to read only the abstract to get the gist of what this specification does.

[the remark this answers is not in the document]
Comment (sullivph) [reply EIPS/eip-6981.md]: Gotcha, I misunderstood your at first. Updated the abstract with the brief technical outline and removed the `Usage Example` section as it seems unnecessary. Thanks!

**Review by xlansmasry — APPROVED** [review]

**Review by SamWilsn — APPROVED** [review]

File: EIPS/eip-6981.md [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+     * @param expiration - If expiration > 0, represents expiration time for the signature.  Otherwise
+     *                     signature does not expire.
+     * @param message    - The keccak256 message which validates the owner, salt, expiration
+     * @param signature  - The signature which validates the owner, salt, expiration
+     *
+     * Emits AccountClaimed event
+     * @return the address of the claimed Account Instance
+     */
+    function claimAccount(
+        address owner,
```
Comment (csilence918) [inline EIPS/eip-6981.md:81]: 0x8A61d49623b6cBEF7C5F54Cb79B5377f8883E1Fd

**Review by csilence918 — COMMENTED** [review]
Aj9AhBigQVQNG7KwgD3WwBB5LP9zDM2w1dNRv2oQhZin is sol ownership addresss to receieve

**Pull request merged**